packages/rancher-istio/charts/values.yaml

```diff
@@ -47,6 +61,7 @@ sidecarInjectorWebhook:
   rewriteAppHTTPProbe: true
 
 global:
+  istioNamespace: cattle-istio-system
```

Comment:
Based on how this is being used to override the default namespace we deploy Istio in, I think that we should call this `istioNamespaceOverride` instead of `istioNamespace`

Reply:
istioNamespace is the istio way to override the default namespace so I propose we leave it to make it easier when comparing with istio documentation. But this does bring up a good question - do we want to fall back to the default istio-system or do we want to enforce the cattle-istio-system which might be a good reason to add that additional override value so it would be {{ <override value> | cattle-istio-system | istio-system }}. allowing it to default to istio-system when istioNamespace isn't set makes the chart more appealing outside of rancher.

Reply:
Hmm I agree with keeping this inline with istio documentation by having `istioNamespace: istio-system`.

I also like the idea of having an additional override value like `istioNamespaceOverride: cattle-istio-system` that overrides the `istioNamespace` value provided in the `values.yaml`. That way a user who wants to use this chart outside of Dashboard can just set `istioNamespaceOverride: ""` if required and fallback to `istioNamespace: istio-system` like they would normally expect without any more changes.

@cbron do you have any thoughts on this?

Reply:
I like that approach too since basically no one installs the standard istio platform outside of istio-system NS. They wouldn't expect it to go to cattle NS. Either partial or default func is fine, I guess partial is cleaner.